src/example-types/facet.js

```diff
@@ -134,5 +135,76 @@ module.exports = {
           }),
         options
       ),
-    })),
+    }))
+
+    let missedValues = _.difference(
+      values,
+      _.map(x => _.toString(x.name), results.options)
+    )
+
+    let getSelectValues = node =>
+      node.isMongoId ? _.map(ObjectID, values) : values
+
+    let matchSelectedValues = node => ({
+      $match: { _id: { $in: getSelectValues(node) } },
+    })
+
+    if (!_.isEmpty(values) && !_.isEmpty(missedValues)) {
```

Comment:
seems redundant since if there are no values there will be no missed values, right? so it can just be `if (!_.isEmpty(missedValues))`